test/e2e-common.sh

```diff
@@ -618,10 +618,10 @@ function overlay_system_namespace() {
 }
 
 function run_ytt() {
-  go_run github.com/vmware-tanzu/carvel-ytt/cmd/ytt@v0.44.1 "$@"
+  go_run github.com/carvel-dev/ytt/cmd/ytt@v0.44.1 "$@"
```

Comment:
go module is still vmware-tanzu

https://github.com/carvel-dev/ytt/blob/develop/go.mod#L1

Reply:
Yes, it was actually that issue https://github.com/golang/go/issues/63684 causing the errors.